XenAdmin/Wizards/PatchingWizard/PlanActions/PoolPatchMapping.cs

```diff
@@ -10,9 +10,25 @@ namespace XenAdmin.Wizards.PatchingWizard.PlanActions
 {
     public class PoolPatchMapping

```

Comment:
Same in this class: you could extend it equality operator overloads etc to avoid similar issues in future.